src/test/java/org/apache/commons/dbcp2/TestPoolingDataSource.java

```diff
@@ -101,9 +101,8 @@ public void testPoolGuardConnectionWrapperEqualsSameDelegate() throws Exception
     public void testPoolGuardConnectionWrapperEqualsReflexive()
         throws Exception {
         final Connection con = ds.getConnection();
-        final Connection con2 = con;
```

Comment:
Don't change there I think. See the title of the method. This makes it obvious.

Reply:
Done